backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -2234,7 +2234,7 @@ void SurfaceSdlGraphicsManager::setMouseCursor(const void *buf, uint w, uint h,
 		const uint numPixels = w * h;
 		const uint inBPP = format->bytesPerPixel;
 
-		Graphics::PixelFormat formatWithAlpha = Graphics::createPixelFormat<8888>();
+		Graphics::PixelFormat formatWithAlpha(4, 8, 8, 8, 8, 16, 8, 0, 24);
```

Comment:
What is the rationale for going away from these convenient templates? Maybe we could just fix the template code if needed?

Reply:
Can't you implement this new functionality inside createPixelFormat(), so that you don't have to replace its usage?

Reply:
Probably not without potentially breaking existing users of `createPixelFormat` that make assumptions about its layout.

If you'd rather I still use a shorthand pixel format, I can update most of these to use `createFormatRGBA32()` instead, which would also help to cut down on the amount of different formats used throughout the codebase.

Reply:
Sounds good! Yep, if it's possible, use this